src/components/PostsList/postsLists.styled.ts

```diff
@@ -1,7 +1,33 @@
 import styled from "styled-components";
 
 export const List = styled('ul')`
+  width: auto;
+  margin: 20px auto;
```

Comment:
I see you are using much hard coded spacings like margin and padding

It is a good practice to use `theme` from `styled-components`

Could you please add `xstyled`? Lets start with spacings and typography!

You can check out the docs [here](https://xstyled.dev/docs/magic-styled-components/)